frontend/src/firebase/actions.ts

```diff
@@ -366,7 +367,68 @@ export const removeSubTask = (
 
 /*
  * --------------------------------------------------------------------------------
- * Section 3: Other Compound Actions
+ * Section 3: Groups Actions
+ * --------------------------------------------------------------------------------
+ */
+
+export const joinGroup = async (
+  groupId: string,
+): Promise<void> => {
```

Comment:
You're probably going to need to take one more argument here: an invite ID.
This way, we can easily query the list of invites via our security rules to ensure the user has permission to join that particular group.